2020-04-22 22:37:05,135 [main] INFO  o.a.a.mft.agent.MFTAgent Starting MFTAgent on shivali with PID 11247 (/home/shivali/Documents/aniruddh-mft/airavata-mft/agent/target/classes started by shivali in /home/shivali/Documents/aniruddh-mft/airavata-mft/agent) []
2020-04-22 22:37:05,137 [main] INFO  o.a.a.mft.agent.MFTAgent No active profile set, falling back to default profiles: default []
2020-04-22 22:37:05,766 [main] WARN  o.s.c.a.AnnotationConfigApplicationContext Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'MFTAgent': Unsatisfied dependency expressed through field 'mftConsulClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mftConsulClient' defined in class path resource [org/apache/airavata/mft/agent/AppConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul []
2020-04-22 22:37:05,771 [main] INFO  o.s.b.a.l.ConditionEvaluationReportLoggingListener 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled. []
2020-04-22 22:37:05,778 [main] ERROR o.s.boot.SpringApplication Application run failed []
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'MFTAgent': Unsatisfied dependency expressed through field 'mftConsulClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mftConsulClient' defined in class path resource [org/apache/airavata/mft/agent/AppConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:639)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:116)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:397)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.airavata.mft.agent.MFTAgent.main(MFTAgent.java:296)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mftConsulClient' defined in class path resource [org/apache/airavata/mft/agent/AppConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:645)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:475)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:636)
	... 18 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:640)
	... 31 common frames omitted
Caused by: com.orbitz.consul.ConsulException: Error connecting to Consul
	at com.orbitz.consul.AgentClient.ping(AgentClient.java:69)
	at com.orbitz.consul.Consul$Builder.build(Consul.java:708)
	at org.apache.airavata.mft.admin.MFTConsulClient.<init>(MFTConsulClient.java:77)
	at org.apache.airavata.mft.agent.AppConfig.mftConsulClient(AppConfig.java:36)
	at org.apache.airavata.mft.agent.AppConfig$$EnhancerBySpringCGLIB$$aee7682a.CGLIB$mftConsulClient$0(<generated>)
	at org.apache.airavata.mft.agent.AppConfig$$EnhancerBySpringCGLIB$$aee7682a$$FastClassBySpringCGLIB$$61b718e2.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.apache.airavata.mft.agent.AppConfig$$EnhancerBySpringCGLIB$$aee7682a.mftConsulClient(<generated>)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 32 common frames omitted
Caused by: java.net.ConnectException: Failed to connect to localhost/127.0.0.1:8500
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:249)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:167)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.orbitz.consul.cache.TimeoutInterceptor.intercept(TimeoutInterceptor.java:53)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:250)
	at okhttp3.RealCall.execute(RealCall.java:93)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:186)
	at com.orbitz.consul.AgentClient.ping(AgentClient.java:62)
	... 45 common frames omitted
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:609)
	at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
	... 67 common frames omitted
2020-04-22 22:39:02,350 [main] INFO  o.a.a.mft.agent.MFTAgent Starting MFTAgent on shivali with PID 11405 (/home/shivali/Documents/aniruddh-mft/airavata-mft/agent/target/classes started by shivali in /home/shivali/Documents/aniruddh-mft/airavata-mft/agent) []
2020-04-22 22:39:02,352 [main] INFO  o.a.a.mft.agent.MFTAgent No active profile set, falling back to default profiles: default []
2020-04-22 22:39:02,876 [main] WARN  o.s.c.a.AnnotationConfigApplicationContext Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'MFTAgent': Unsatisfied dependency expressed through field 'mftConsulClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mftConsulClient' defined in class path resource [org/apache/airavata/mft/agent/AppConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul []
2020-04-22 22:39:02,881 [main] INFO  o.s.b.a.l.ConditionEvaluationReportLoggingListener 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled. []
2020-04-22 22:39:02,887 [main] ERROR o.s.boot.SpringApplication Application run failed []
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'MFTAgent': Unsatisfied dependency expressed through field 'mftConsulClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mftConsulClient' defined in class path resource [org/apache/airavata/mft/agent/AppConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:639)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:116)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:397)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.airavata.mft.agent.MFTAgent.main(MFTAgent.java:296)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mftConsulClient' defined in class path resource [org/apache/airavata/mft/agent/AppConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:645)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:475)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:636)
	... 18 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.airavata.mft.admin.MFTConsulClient]: Factory method 'mftConsulClient' threw exception; nested exception is com.orbitz.consul.ConsulException: Error connecting to Consul
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:640)
	... 31 common frames omitted
Caused by: com.orbitz.consul.ConsulException: Error connecting to Consul
	at com.orbitz.consul.AgentClient.ping(AgentClient.java:69)
	at com.orbitz.consul.Consul$Builder.build(Consul.java:708)
	at org.apache.airavata.mft.admin.MFTConsulClient.<init>(MFTConsulClient.java:77)
	at org.apache.airavata.mft.agent.AppConfig.mftConsulClient(AppConfig.java:36)
	at org.apache.airavata.mft.agent.AppConfig$$EnhancerBySpringCGLIB$$bd58b00a.CGLIB$mftConsulClient$0(<generated>)
	at org.apache.airavata.mft.agent.AppConfig$$EnhancerBySpringCGLIB$$bd58b00a$$FastClassBySpringCGLIB$$42eb663f.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.apache.airavata.mft.agent.AppConfig$$EnhancerBySpringCGLIB$$bd58b00a.mftConsulClient(<generated>)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 32 common frames omitted
Caused by: java.net.ConnectException: Failed to connect to localhost/127.0.0.1:8500
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:249)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:167)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.orbitz.consul.cache.TimeoutInterceptor.intercept(TimeoutInterceptor.java:53)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:250)
	at okhttp3.RealCall.execute(RealCall.java:93)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:186)
	at com.orbitz.consul.AgentClient.ping(AgentClient.java:62)
	... 45 common frames omitted
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:609)
	at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
	... 67 common frames omitted
2020-04-22 22:39:34,125 [main] INFO  o.a.a.mft.agent.MFTAgent Starting MFTAgent on shivali with PID 11659 (/home/shivali/Documents/aniruddh-mft/airavata-mft/agent/target/classes started by shivali in /home/shivali/Documents/aniruddh-mft/airavata-mft/agent) []
2020-04-22 22:39:34,128 [main] INFO  o.a.a.mft.agent.MFTAgent No active profile set, falling back to default profiles: default []
2020-04-22 22:39:34,847 [main] INFO  o.a.a.mft.agent.MFTAgent Started MFTAgent in 0.918 seconds (JVM running for 1.331) []
2020-04-22 22:39:34,848 [main] INFO  o.a.a.mft.agent.MFTAgent Starting Agent agent0 []
2020-04-22 22:39:34,965 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock true []
2020-04-22 22:39:34,966 [main] INFO  o.a.a.mft.agent.MFTAgent Successfully connected to consul with session id ba8be593-8ffb-5b34-f903-0b1fa1696758 []
2020-04-22 22:39:52,912 [main] INFO  o.a.a.mft.agent.MFTAgent Starting MFTAgent on shivali with PID 11726 (/home/shivali/Documents/aniruddh-mft/airavata-mft/agent/target/classes started by shivali in /home/shivali/Documents/aniruddh-mft/airavata-mft/agent) []
2020-04-22 22:39:52,914 [main] INFO  o.a.a.mft.agent.MFTAgent No active profile set, falling back to default profiles: default []
2020-04-22 22:39:53,523 [main] INFO  o.a.a.mft.agent.MFTAgent Started MFTAgent in 0.795 seconds (JVM running for 1.214) []
2020-04-22 22:39:53,524 [main] INFO  o.a.a.mft.agent.MFTAgent Starting Agent agent0 []
2020-04-22 22:39:53,630 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:39:53,630 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:39:58,663 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:39:58,664 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:03,699 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:03,699 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:04,341 [main] INFO  o.a.a.mft.agent.MFTAgent Starting MFTAgent on shivali with PID 11777 (/home/shivali/Documents/aniruddh-mft/airavata-mft/agent/target/classes started by shivali in /home/shivali/Documents/aniruddh-mft/airavata-mft/agent) []
2020-04-22 22:40:04,342 [main] INFO  o.a.a.mft.agent.MFTAgent No active profile set, falling back to default profiles: default []
2020-04-22 22:40:05,018 [main] INFO  o.a.a.mft.agent.MFTAgent Started MFTAgent in 0.877 seconds (JVM running for 1.318) []
2020-04-22 22:40:05,019 [main] INFO  o.a.a.mft.agent.MFTAgent Starting Agent agent0 []
2020-04-22 22:40:05,142 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:05,142 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:08,733 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:08,734 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:10,185 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:10,186 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:13,760 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:13,760 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:15,201 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:15,201 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:18,786 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:18,787 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:20,235 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:20,235 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:23,814 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:23,815 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:25,256 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:25,257 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:28,830 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:28,830 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:30,279 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:30,280 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:33,847 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:33,847 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:35,295 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:35,295 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:38,876 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:38,876 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:40,315 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:40,316 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:43,898 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:43,899 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:45,336 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:45,337 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:48,904 [main] INFO  o.s.b.a.l.ConditionEvaluationReportLoggingListener 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled. []
2020-04-22 22:40:48,913 [main] ERROR o.s.boot.SpringApplication Application run failed []
java.lang.IllegalStateException: Failed to execute CommandLineRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:787)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:768)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:322)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.airavata.mft.agent.MFTAgent.main(MFTAgent.java:296)
Caused by: java.lang.Exception: Failed to connect to the cluster
	at org.apache.airavata.mft.agent.MFTAgent.start(MFTAgent.java:279)
	at org.apache.airavata.mft.agent.MFTAgent.run(MFTAgent.java:290)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:784)
	... 5 common frames omitted
2020-04-22 22:40:50,356 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:50,356 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:40:55,376 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:40:55,376 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:41:00,380 [main] INFO  o.s.b.a.l.ConditionEvaluationReportLoggingListener 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled. []
2020-04-22 22:41:00,408 [main] ERROR o.s.boot.SpringApplication Application run failed []
java.lang.IllegalStateException: Failed to execute CommandLineRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:787)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:768)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:322)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.airavata.mft.agent.MFTAgent.main(MFTAgent.java:296)
Caused by: java.lang.Exception: Failed to connect to the cluster
	at org.apache.airavata.mft.agent.MFTAgent.start(MFTAgent.java:279)
	at org.apache.airavata.mft.agent.MFTAgent.run(MFTAgent.java:290)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:784)
	... 5 common frames omitted
2020-04-22 22:44:57,736 [main] INFO  o.a.a.mft.agent.MFTAgent Starting MFTAgent on shivali with PID 12523 (/home/shivali/Documents/aniruddh-mft/airavata-mft/agent/target/classes started by shivali in /home/shivali/Documents/aniruddh-mft/airavata-mft/agent) []
2020-04-22 22:44:57,738 [main] INFO  o.a.a.mft.agent.MFTAgent No active profile set, falling back to default profiles: default []
2020-04-22 22:44:58,339 [main] INFO  o.a.a.mft.agent.MFTAgent Started MFTAgent in 0.782 seconds (JVM running for 1.347) []
2020-04-22 22:44:58,340 [main] INFO  o.a.a.mft.agent.MFTAgent Starting Agent agent0 []
2020-04-22 22:44:58,477 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock true []
2020-04-22 22:44:58,478 [main] INFO  o.a.a.mft.agent.MFTAgent Successfully connected to consul with session id c8428d55-7694-1e9b-19d5-5bc6a7ed91ab []
2020-04-22 22:45:12,692 [RMI TCP Connection(2)-127.0.0.1] INFO  o.s.b.a.SpringApplicationAdminMXBeanRegistrar$SpringApplicationAdmin Application shutdown requested. []
2020-04-22 22:45:52,458 [main] INFO  o.a.a.mft.agent.MFTAgent Starting MFTAgent on shivali with PID 12618 (/home/shivali/Documents/aniruddh-mft/airavata-mft/agent/target/classes started by shivali in /home/shivali/Documents/aniruddh-mft/airavata-mft/agent) []
2020-04-22 22:45:52,459 [main] INFO  o.a.a.mft.agent.MFTAgent No active profile set, falling back to default profiles: default []
2020-04-22 22:45:53,253 [main] INFO  o.a.a.mft.agent.MFTAgent Started MFTAgent in 1.004 seconds (JVM running for 1.457) []
2020-04-22 22:45:53,254 [main] INFO  o.a.a.mft.agent.MFTAgent Starting Agent agent0 []
2020-04-22 22:45:53,381 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:45:53,382 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:45:58,411 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock false []
2020-04-22 22:45:58,412 [main] INFO  o.a.a.mft.agent.MFTAgent Retrying to connect to consul []
2020-04-22 22:46:03,464 [main] INFO  o.a.a.mft.agent.MFTAgent Acquired lock true []
2020-04-22 22:46:03,464 [main] INFO  o.a.a.mft.agent.MFTAgent Successfully connected to consul with session id 655288a2-1f79-ae5c-f575-d8a1e5389b49 []
2020-04-22 23:39:49,057 [OkHttp http://localhost:8500/...] ERROR c.o.consul.cache.ConsulCache Error getting response from consul for keyvalue "mft/agents/messages/agent0", will retry in 10000 MILLISECONDS []
java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at okio.Okio$2.read(Okio.java:140)
	at okio.AsyncTimeout$2.read(AsyncTimeout.java:237)
	at okio.RealBufferedSource.indexOf(RealBufferedSource.java:354)
	at okio.RealBufferedSource.readUtf8LineStrict(RealBufferedSource.java:226)
	at okhttp3.internal.http1.Http1Codec.readHeaderLine(Http1Codec.java:215)
	at okhttp3.internal.http1.Http1Codec.readResponseHeaders(Http1Codec.java:189)
	at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:88)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.orbitz.consul.cache.TimeoutInterceptor.intercept(TimeoutInterceptor.java:53)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:250)
	at okhttp3.RealCall$AsyncCall.execute(RealCall.java:201)
	at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-04-22 23:39:51,454 [pool-3-thread-1] WARN  o.a.a.mft.agent.MFTAgent Errored while renewing the session []
com.orbitz.consul.ConsulException: Consul request failed
	at com.orbitz.consul.util.Http.executeCall(Http.java:52)
	at com.orbitz.consul.util.Http.extract(Http.java:31)
	at com.orbitz.consul.SessionClient.renewSession(SessionClient.java:80)
	at com.orbitz.consul.SessionClient.renewSession(SessionClient.java:69)
	at org.apache.airavata.mft.agent.MFTAgent.lambda$connectAgent$9(MFTAgent.java:210)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.net.ConnectException: Failed to connect to localhost/127.0.0.1:8500
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:249)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:167)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.orbitz.consul.cache.TimeoutInterceptor.intercept(TimeoutInterceptor.java:53)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:250)
	at okhttp3.RealCall.execute(RealCall.java:93)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:186)
	at com.orbitz.consul.util.Http.executeCall(Http.java:49)
	... 10 common frames omitted
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:609)
	at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
	... 32 common frames omitted
2020-04-22 23:39:51,457 [pool-3-thread-1] ERROR o.a.a.mft.agent.MFTAgent Can not renew session as it is expired []
2020-04-22 23:39:51,458 [pool-3-thread-1] INFO  o.a.a.mft.agent.MFTAgent Stopping Agent agent0 []
2020-04-22 23:39:51,459 [main] INFO  o.a.a.mft.agent.MFTAgent Agent exited []
